getting_started/hello_world.md

```diff
@@ -22,7 +22,7 @@ content:
 
 ```bash
 #!/bin/bash
-#SBATCH --cpus=1
+#SBATCH --N=1
```

Comment:
`--N` is not a valid slurm option. The correct syntax would be `--nodes` or `-N`. However, I think we should leave it as `-n`. `-n` specifies the number of tasks, and one can add `--cpus-per-task` or `-c` in order to specify the number of cpus for each task. Slurm has a default of 1 cpu per task, so just specifying `-n` here is valid.